samples/subsys/fs/littlefs/README.rst

```diff
@@ -0,0 +1,69 @@
+.. _sample_subsys_fs_littlefs:
+
+littlefs Filesystem Sample Application
+######################################
+
+Overview
+********
+
+This sample app demonstrates use of Zephyr's :ref:`file system
+API<file_system>` over `littlefs`_, using a file system with a file that
+counts the number of times the system has booted.  Other information
+about the file system is also displayed.
+
+.. _littlefs:
+   https://github.com/ARMmbed/littlefs
+
+Requirements
+************
+
+The storage partition area will be used for the file system; see
+:ref:`flash_partitions`.  If that area does not already have a
+compatible littlefs file system its contents will be replaced by an
+empty file system.  You will see diagnostics like this::
+
+   [00:00:00.010,192] <inf> littlefs: LittleFS version 2.0, disk version 2.0
+   [00:00:00.010,559] <err> littlefs: Corrupted dir pair at 0 1
+   [00:00:00.010,559] <wrn> littlefs: can't mount (LFS -84); formatting
+
+The error and warning are normal for a new file system.
+
+After the file system is mounted you'll also see::
+
+   [00:00:00.182,434] <inf> littlefs: filesystem mounted!
+   [00:00:00.867,034] <err> fs: failed get file or dir stat (-2)
+
+This error is also normal for Zephyr not finding a file (the boot count,
+in this case).
+
+Building and Running
+********************
+
+This example should work on any board that defines a ``storage``
```

Comment:
Yes.  https://docs.zephyrproject.org/latest/guides/bluetooth/bluetooth-arch.html#persistent-storage has it, so it should be "storage" rather than `storage`.  OK?

Reply:
I found that in my grepping, (and found a bunch of .overlay files defining partitions with the "storage" label, but I didn't find anything specific of how to define a "storage" partition.  Oh well, maybe it's obvious to those that know.

Reply:
Please see whether the rework reads better.  The use in Requirements is more complete; the use here is a basic reference.